README.md

```diff
@@ -58,8 +58,9 @@ After installation you can directly start using it. You can do this by either us
 | Show all bookmarks (toggle)                     | `ma`        | `:BookmarkShowAll`           |
 | Clear bookmarks in current buffer only          | `mc`        | `:BookmarkClear`             |
 | Clear bookmarks in all buffers                  | `mx`        | `:BookmarkClearAll`          |
-| Move up bookmark at current line                | `mkk`       | `:BookmarkMoveUp`            |
-| Move down bookmark at current line              | `mjj`       | `:BookmarkMoveDown`          |
+| Move up bookmark at current line                | `[count]mkk`| `:BookmarkMoveUp`            |
+| Move down bookmark at current line              | `[count]mjj`| `:BookmarkMoveDown`          |
+| Move bookmark at current line to another line   |             | `:BookmarkMoveToLine <LINE>` |
```

Comment:
Leaving default unmapped for the sake of simplicity. Registering a default shortcut sticking to the convention laid out by `s:register_mapping` would mean further changes to `function BookmarkModify` to make all arguments optional, just like `function BookmarkAnnotate(...)`. The only solution then to keep consistent with the existing convention at the cost of being a bit less DRY, would be to create a new `function BookmarkMoveToLine(...)` essentially duplicating `BookmarkModify`'s logic.